Assets/MixedRealityToolkit.Services/InputSimulation/SimulatedHand.cs

```diff
@@ -8,6 +8,71 @@
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
+    /// <summary>
+    /// Snapshot of simulated hand data.
+    /// </summary>
+    [System.Serializable]
+    public class SimulatedHandData
+    {
+        private static readonly int jointCount = Enum.GetNames(typeof(TrackedHandJoint)).Length;
```

Comment:
>Enum.GetNames(typeof(TrackedHandJoint)).Length [](start = 49, length = 46)

This will include the 'None' enum value, in case that's an issue.

Reply:
The purpose of this is to get the necessary array size, so we can use `(int)TrackedHandJoint` as an index. The `None` item is pretty much unused, so not a problem (apart from minor size overhead, not worth optimizing at this point).